### PR TITLE
Convert tutorial to md format

### PR DESCRIPTION
**Pull request by hyanwong**
This also includes some larger sample data files so that we don't have to do the large inference in the notebook itself (takes about 17 mins)

(edit - now these large files have been removed)

### CODE REVIEW

**Comment by hyanwong** [comment]
Actually, I just worked around it by saying

> Below, for simplicity we use the smaller dataset that was simulated and
> inferred within the the notebook, but exactly the same code will work with
> the larger version run from the cli simulation

**Comment by benjeffery** [comment]
> Thanks for looking @benjeffery - I'm having difficulty finding a version of cyvcf2 that can be successfully placed in `requirements/CI-docs/requirements.txt`. How do I do that?

How do you mean? What trouble?

File: docs/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,694 @@
...
+(sec_tutorial_read_vcf)=
+
+### Reading a VCF
+
+A common way to store genetic variation data is as a VCF (Variant Call Format) file.
+The [CYVCF2 library](https://github.com/brentp/cyvcf2) provides a relatively fast
+Python interface for reading VCF files. The following script demonstrates how to infer a
+tree sequence from a simple VCF file. In particular the function below named
+`add_diploid_sites` illustrates how to iterate over the variants in a `CYVCF2.VCF`
+object and add them to a `tsinfer`` sample data file. This particular script assumes
```
Comment (jeromekelleher) [inline docs/tutorial.md:419]:
```suggestion
object and add them to a ``tsinfer`` sample data file. This particular script assumes
```

**Review by jeromekelleher — APPROVED** [review]
LGTM, spotted one typo

File: docs/tutorial.md (hunk 1 above)
Comment (hyanwong) [reply docs/tutorial.md]: New version pushed. Thanks for spotting the typo @jeromekelleher

**Comment by hyanwong** [comment]
Currently failing because of ubuntu 18.04 being deprecated. @benjeffery will fix later.

File: requirements/CI-docs/requirements.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
+cvycf2==0.30.00
```
Comment (benjeffery) [inline requirements/CI-docs/requirements.txt:1]:
```suggestion
cyvcf2==0.30.00
```

**Review by benjeffery — APPROVED** [review]
CI is failing due to a typo.

File: requirements/CI-docs/requirements.txt (hunk 2 above)
Comment (hyanwong) [reply requirements/CI-docs/requirements.txt:1]: Doh, sorry

**Comment by benjeffery** [comment]
Think you need 0.30.18

**Comment by hyanwong** [comment]
Really sorry about this Ben - kicking myself that I didn't spot the typo!

**Comment by hyanwong** [comment]
The problem is now with accessing the URL:

```
OSError                                   Traceback (most recent call last)
[177](https://github.com/tskit-dev/tsinfer/actions/runs/3283391161/jobs/5408044963#step:9:178)
Cell In [9], line 49
[178](https://github.com/tskit-dev/tsinfer/actions/runs/3283391161/jobs/5408044963#step:9:179)
     46 # URL for the VCF
[179](https://github.com/tskit-dev/tsinfer/actions/runs/3283391161/jobs/5408044963#step:9:180)
     47 url = "https://github.com/tskit-dev/tsinfer/raw/main/docs/_static/P_dom_chr24_phased.vcf.gz"
[180](https://github.com/tskit-dev/tsinfer/actions/runs/3283391161/jobs/5408044963#step:9:181)
---> 49 vcf = cyvcf2.VCF(url)
[181](https://github.com/tskit-dev/tsinfer/actions/runs/3283391161/jobs/5408044963#step:9:182)
     50 with tsinfer.SampleData(
[182](https://github.com/tskit-dev/tsinfer/actions/runs/3283391161/jobs/5408044963#step:9:183)
     51     path="P_dom_chr24_phased.samples", sequence_length=chromosome_length(vcf)
[183](https://github.com/tskit-dev/tsinfer/actions/runs/3283391161/jobs/5408044963#step:9:184)
     52 ) as samples:
[184](https://github.com/tskit-dev/tsinfer/actions/runs/3283391161/jobs/5408044963#step:9:185)
     53     add_diploid_sites(vcf, samples)
[185](https://github.com/tskit-dev/tsinfer/actions/runs/3283391161/jobs/5408044963#step:9:186)

[186](https://github.com/tskit-dev/tsinfer/actions/runs/3283391161/jobs/5408044963#step:9:187)
File /opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/cyvcf2/cyvcf2.pyx:258, in cyvcf2.cyvcf2.VCF.__init__()
[187](https://github.com/tskit-dev/tsinfer/actions/runs/3283391161/jobs/5408044963#step:9:188)

[188](https://github.com/tskit-dev/tsinfer/actions/runs/3283391161/jobs/5408044963#step:9:189)
File /opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/cyvcf2/cyvcf2.pyx:190, in cyvcf2.cyvcf2.HTSFile._open_htsfile()
[189](https://github.com/tskit-dev/tsinfer/actions/runs/3283391161/jobs/5408044963#step:9:190)

[190](https://github.com/tskit-dev/tsinfer/actions/runs/3283391161/jobs/5408044963#step:9:191)
OSError: Error opening https://github.com/tskit-dev/tsinfer/raw/main/docs/_static/P_dom_chr24_phased.vcf.gz
[191](https://github.com/tskit-dev/tsinfer/actions/runs/3283391161/jobs/5408044963#step:9:192)
OSError: Error opening https://github.com/tskit-dev/tsinfer/raw/main/docs/_static/P_dom_chr24_phased.vcf.gz
[192](https://github.com/tskit-dev/tsinfer/actions/runs/3283391161/jobs/5408044963#step:9:193)
```

**Comment by benjeffery** [comment]
> Really sorry about this Ben - kicking myself that I didn't spot the typo!

No biggie, I am the typo king.

**Comment by benjeffery** [comment]
> The problem is now with accessing the URL where the VCF is stored

I don't think `cyvcf2` accepts URLs?

**Comment by hyanwong** [comment]
> > The problem is now with accessing the URL where the VCF is stored
> 
> I don't think `cyvcf2` accepts URLs?

Weird. It does for me and all compiles fine on my machine. You are right that it's not obviously documented any more, though. Let me investigate. I guess we could provide the direct file link here anyway. It was only a URL to allow people to run it in their own notebooks.

**Comment by hyanwong** [comment]
Hooray. Passes now. Thanks @benjeffery

**Pull request merged**